features/cellular/UNITTESTS/at/at_cellularinformation/test_at_cellularinformation.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2018, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "CppUTest/TestHarness.h"
```

Comment:
Not yet. There is ongoing discussion around unittesting in general. CppUTest is one of the options, but we might switch to something else as well. When we have decided the mbed OS wide unittest tool, we need to document that for the whole mbed OS, not just for Cellular. There is a run_tests script in UNITTESTS folder and I think it is quite self-explanatory. Do you think we should add "readme" or similar under UNITTESTS folder?

Reply:
just want to understand why UNITTEST folder is also part of this PR . If it is added, then it needs to be documented how it is used, otherwise it can ber considered "dead" code - why it is here then?

Reply:
Product decision. README.md updated with instructions